script/server

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")/.."
```

Comment:
This is Mac OS X specific command and it will fail on heroku or any other linux server.